.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm run lint
```

Comment:
This feels a _little_ heavy handed since CI should fail anyways, right?

Reply:
Fair enough, I'll just add this for myself 😆